src/main/java/roomescape/global/jwt/JwtProvider.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.global.jwt;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import roomescape.domain.member.domain.Role;
+import roomescape.global.auth.AuthUser;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+
+@Component
+public class JwtProvider {
+
+    public static final String NAME = "name";
+    public static final String ROLE = "role";
+
+    @Value("${jwt.secret}")
+    private String secretKey;
+
+    @Value("${jwt.expiration}")
+    private int expiration;
```

Comment:
[제안]
단위까지 명시해주시면 다른 개발자들이 사용할 때 편리합니다.

```suggestion
    @Value("${jwt.expiration}")
    private int expirationSeconds;
```

Reply:
좋은 팁 감사합니다!! 👍